indigo_api/management/commands/migrate_taxonomies.py

```diff
@@ -0,0 +1,21 @@
+from django.core.management import BaseCommand
+from indigo_api.models import TaxonomyTopic, TaxonomyVocabulary, Work
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        for t in TaxonomyVocabulary.objects.all().iterator():
+            root = TaxonomyTopic.add_root(name=t.title)
+            for vocabulary in t.topics.all():
+                if vocabulary.level_1:
+                    child = TaxonomyTopic.add_child(root, name=vocabulary.level_1)
+                    if vocabulary.level_2:
+                        TaxonomyTopic.add_child(child, name=vocabulary.level_2)
+
+        for work in Work.objects.all().iterator():
+            for taxonomy in work.taxonomies.all():
+                topic = taxonomy.title
+                level_1 = taxonomy.level_1
+                level_2 = taxonomy.level_2
+                topics = TaxonomyTopic.objects.filter(name__in=[topic, level_1, level_2])
+                work.taxonomy_topics.add(*topics)
```

Comment:
@longhotsummer, @goose-life  Here is the migration command. I still have the two for loops, it seems easier this way. I have the TaxonomyVocabulary object as the root here where I then loop through all related Vocabulary topic objects to build the tree.